pss/pss_test.go

```diff
@@ -76,22 +78,23 @@ func initTest() {
 			h := log.CallerFileHandler(hf)
 			log.Root().SetHandler(h)
 
-			cryptoUtils = NewCryptoUtils()
+			cryptoUtils = crypto.NewCryptoUtils()
 
 			pssprotocols = make(map[string]*protoCtrl)
 		},
 	)
 }
 
-// test that topic conversion functions give predictable results
-func TestTopic(t *testing.T) {
+// test that API topic conversion functions give predictable results
+
+func TestAPITopic(t *testing.T) {
 
 	api := &API{}
 
 	topicstr := strings.Join([]string{PingProtocol.Name, strconv.Itoa(int(PingProtocol.Version))}, ":")
```

Comment:
This is an ugly relic, indeed. Maybe we can just `fmt.Sprintf` this? I don't think `strings.Join` and `strconv` is used like this in _any_ other test in swarm.

Reply:
I am not refactoring these tests, just carving `PssMessage` out.